src/reducers/fileTree.tsx

```diff
@@ -157,6 +159,74 @@ export const buildFileTree = (version: Version): FileTree => {
   };
 };
 
+type GetRelativePathParams = {
+  _log: typeof log;
+  currentPath: string;
+  pathList: string[];
+  position: 'previous' | 'next';
+};
+
+export const getRelativePath = ({
+  _log = log,
+  currentPath,
+  pathList,
+  position,
+}: GetRelativePathParams): string | void => {
+  const currentIndex = pathList.indexOf(currentPath);
+  if (!currentIndex) {
+    _log.debug(`Cannot find ${currentPath} in pathList: ${pathList}`);
+    return undefined;
+  }
+  const newIndex =
+    position === 'previous' ? currentIndex - 1 : currentIndex + 1;
+  const validIndex = newIndex >= 0 && pathList.length >= newIndex;
+  if (!validIndex) {
+    _log.debug(`There is not a ${position} file wrt ${currentPath}`);
```

Comment:
Could you do this without an abbreviation? I know it was just a draft PR but there ya go 😄